app/experimenter/experiments/serializers/recipe.py

```diff
@@ -336,6 +351,9 @@ def get_filter_object(self, obj):
         if obj.countries.count():
             filter_objects.append(FilterObjectCountrySerializer(obj).data)
 
+        if len(obj.platforms) <= 2:
```

Comment:
If you move the list of all platforms on to constants, then you can do `len(PLATFORMS)`  here instead of hardcoding `2`.  Likelihood of adding platforms is small, but still if we ever change that list then this will  update accordingly.